app/server/models/user.py

```diff
@@ -425,21 +426,32 @@ def decode_single_use_JWS(cls, token, required_type):
 
     def save_password_reset_token(self, password_reset_token):
         # make a "clone" of the existing token list
-        self.clear_expired_tokens()
-        current_reset_tokens = self.password_reset_tokens[:]
-        current_reset_tokens.append(password_reset_token)
+        self.clear_expired_password_reset_tokens()
+        current_password_reset_tokens = self.password_reset_tokens[:]
+        current_password_reset_tokens.append(password_reset_token)
         # set db value
-        self.password_reset_tokens = current_reset_tokens
+        self.password_reset_tokens = current_password_reset_tokens
+
+    def save_pin_reset_token(self, pin_reset_token):
```

Comment:
this doesn't seem to be being used either right now. it'll be used in the manual admin reset, but a new user should also be set as resetting pin right? otherwise they wouldn't go down the initial pin setting? or am i mistaken